nixos/modules/services/web-servers/traefik.nix

```diff
@@ -64,6 +64,12 @@ in {
       '';
     };
 
+    group = mkOption {
+      default = "docker";
```

Comment:
Should this be the default? 
Running as `docker` is equivalent to running as root, because you can get root access by launching privileged containers.

Reply:
ah yes, sorry. That was meant to be traefik. Good catch